src/test/node/node-24.spec.ts

```diff
@@ -0,0 +1,17 @@
+// test/node/node-24.spec.ts
+
+import { strict as assert } from 'node:assert';
+import { describe, it } from 'node:test';
+
+await (process.version < 'v24' ? describe.skip : describe)('node-24', () => {
+  it('URLPattern exists as a global', async () => {
+    assert.equal(typeof URLPattern, 'function');
+  });
+
+  // 24.2+
+  it('import.meta.main is available', async () => {
```

Comment:
it fails on node v24.0.0, should we make this test conditional?

Reply:
Since it's only ever being tested on the latest versions of 22.x/24.x (due to the way GitHub actions work) I haven't bothered with that in the past.  I think it's fair to assume anyone running tests on this should be up-to-date, and if this reminds them to update their local node version, probably for the best. :)